forge/forge/op/eval/forge/pooling.py

```diff
@@ -478,15 +478,20 @@ def decompose(type, attr, dc, inputs):
         # If global average
         if y == kH and x == kW and ((stride[0] == kH and stride[1] == kW) or all(pad == 0 for pad in padding)):
             if channel_last:
-                result = dc.op("reshape", [activations], (w, 1, y * x, cin))
-                result = dc.op("reduce_avg", [result], (-2,))
-                result = dc.op("reshape", [result], (w, 1, 1, cin))
+                # for reshape use dc.op_with_named_attrs
```

Comment:
Do we need this comment? In the future, we should transfer to using named attributes for each op :))

Reply:
Yeah it is unnecessary...